### PR TITLE
Hotfix/review state

### PR DESCRIPTION
**Pull request by sorja**
Includes also commit for using `stateKey`

### CODE REVIEW

File: webapp/loggedin/review/reviewState.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ const keys = {
 const getState = R.prop(stateKey)
 
 // === READ
-export const getOpenThread = R.pipe(getState, R.propOr({ target: [], section: '' }, keys.openThread))
+// openThread = { target: [], section: '' },
+export const getOpenThread = R.pipe(getState, R.prop( keys.openThread))
+export const getOpenThreadTarget = R.pipe(getState, R.pathOr([], ['openThread', 'target']))
```
Comment (minotogna) [inline webapp/loggedin/review/reviewState.js]: can you try? ```export const getOpenThreadTarget = R.pipe(getOpenThread, R.propOr([], 'target'))```

File: webapp/tableWithOdp/tableWithOdp.js [hunk 2]
```diff
@@ -352,7 +352,7 @@ const renderFieldRow = ({ row, countryIso, fra, save, saveMany, pasteUpdate, row
   const validator = row.validator || alwaysOkValidator
   return <tr
     key={field}
-    className={`${openCommentThread && R.isEmpty(R.difference(openCommentThread.target, [field])) ? 'fra-row-comments__open' : ''}`}>
+    className={`${openCommentThread && openCommentThread.target && R.isEmpty(R.difference(openCommentThread.target, [field])) ? 'fra-row-comments__open' : ''}`}>
```
Comment (minotogna) [inline webapp/tableWithOdp/tableWithOdp.js]: can you check where this is passed? it should use the ```getOpenThreadTarget```

Comment (sorja) [reply webapp/tableWithOdp/tableWithOdp.js]: Leaving this as is for now

File: webapp/loggedin/review/reducer.js [hunk 3]
```diff
@@ -10,30 +8,35 @@ import {
...
+  [issuePostCommentCompleted]: (state, { status }) => ReviewState.assocStatus(status)(state),
+  [issueRetrieveCommentsStarted]: (state, { status }) => ReviewState.assocStatus(status)(state),
+
+  [issueRetrieveCommentsCompleted]: (state, { target, issue }) =>
+    ReviewState.assocDynamicTarget(target)(
+      {
+        ...ReviewState.getDynamicTarget(target),
+        issue
+      }
+    )(state),
```
Comment (minotogna) [inline webapp/loggedin/review/reducer.js]: from what i see, the target should be merged . assocDynamicTarget accepts only one target

File: webapp/loggedin/review/reducer.js [hunk 4]
```diff
@@ -10,30 +8,35 @@ import {
...
+  [issueGetSummaryCompleted]: (state, { target, issuesCount, lastCommentUserId, issueStatus, hasUnreadIssues }) =>
+    ReviewState.assocDynamicTarget(target)(
+      {
+        ...ReviewState.getDynamicTarget(target),
+        issuesCount,
+        lastCommentUserId,
+        issueStatus,
+        hasUnreadIssues
+      }
+    )(state),
```
Comment (minotogna) [inline webapp/loggedin/review/reducer.js]: same as above

File: webapp/loggedin/review/reducer.js [hunk 5]
```diff
@@ -10,30 +8,35 @@ import {
...
+
+  [issueOpenCommentThread]: (state, { target, section, title }) =>
+    ReviewState.assocOpenThread({ target, section, name: title })(state),
+  [issueCloseCommentThread]: state => ReviewState.omitOpenThread()(state),
```
Comment (minotogna) [inline webapp/loggedin/review/reducer.js]: isn't this ReviewState.omitOpenThread(state) ?

**Pull request merged**